portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/cpdocumentsandmedia/CPDocumentsandmedia.testcase

```diff
@@ -55,47 +55,10 @@
 		</execute>
 	</command>
 
-	<command name="AddDocument" priority="4">
```

Comment:
@anthony-chu weird, it looks like BChan merged it [here](https://github.com/brianchandotcom/liferay-portal/pull/48163).  I'll look at it some more.